resources/definitions/adimlab_base.def.json

```diff
@@ -0,0 +1,268 @@
+{
+    "name": "ADIMLab Base Printer",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": false,
+        "author": "James Radko",
+        "manufacturer": "ADIMLab",
+        "file_formats": "text/x-gcode",
+        "first_start_actions": ["MachineSettingsAction"],
+
+        "machine_extruder_trains": {
+            "0": "adimlab_base_extruder_0"
+        },
+
+        "has_materials": true,
+        "has_variants": true,
+        "has_machine_": true,
+        "variants_name": "Nozzle Size",
+
+        "preferred_variant_name": "0.4mm Nozzle",
+        "preferred_quality_type": "standard",
+        "preferred_material": "generic_pla",
+        "exclude_materials": [
+            "Vertex_Delta_ABS",
+            "Vertex_Delta_PET",
+            "Vertex_Delta_PLA",
+            "Vertex_Delta_TPU",
+            "chromatik_pla",
+            "dsm_arnitel2045_175",
+            "dsm_novamid1070_175",
+            "fabtotum_abs",
+            "fabtotum_nylon",
+            "fabtotum_pla",
+            "fabtotum_tpu",
+            "fiberlogy_hd_pla",
+            "filo3d_pla",
+            "filo3d_pla_green",
+            "filo3d_pla_red",
+            "generic_abs",
+            "generic_bam",
+            "generic_cffcpe",
+            "generic_cffpa",
+            "generic_cpe",
+            "generic_cpe_plus",
+            "generic_gffcpe",
+            "generic_gffpa",
+            "generic_hips",
+            "generic_nylon",
+            "generic_pc",
+            "generic_petg",
+            "generic_pla",
+            "generic_pp",
+            "generic_pva",
+            "generic_tough_pla",
+            "generic_tpu",
+            "imade3d_petg_green",
+            "imade3d_petg_pink",
+            "imade3d_pla_green",
+            "imade3d_pla_pink",
+            "innofill_innoflex60_175",
+            "octofiber_pla",
+            "polyflex_pla",
+            "polymax_pla",
+            "polyplus_pla",
+            "polywood_pla",
+            "structur3d_dap100silicone",
+            "tizyx_abs",
+            "tizyx_pla",
+            "tizyx_pla_bois",
+            "ultimaker_abs_black",
```

Comment:
There is no need to add all these materials, since they are 2.85mm (and thus won't be shown)